src/services/app-api.ts

```
@@ -0,0 +1,13 @@
+    static getUserById(id: string) {
+        return fetch('/api/peanut/user/get-user-id', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                accountIdentifier: id,
+            }),
+        }).then((res) => res.json())
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)